run_llm/benchmark_set_up.py

```diff
@@ -0,0 +1,54 @@
+import json
+import numpy as np
+
+from pathlib import Path
+from abc import ABC, abstractmethod
+from typing import Optional
+
+
+class Benchmark(ABC):
+    data: dict
+    prompt: str
+    labels: Optional[str] = None
+    meshes: Optional[str] = None
+
+    @abstractmethod
+    def get_ground_truth(self):
+        raise NotImplementedError
```

Comment:
No need to raise an `NotImplementedError` here. If not implemented you will get a `TypeError at instantiation time